### ISSUE TITLE
About train

### ISSUE THREAD

**Issue by Bruce-WangGF** [yuhuUSTC/FSDGN#10]
Because I don't very familiar with Linux，I create a new project based Windows, But the Losses have not declined.
I check the Experiments and Loss section, it doesn't seem to have some other operations.
so I want to know whether run the train.py or train.sh directly?

**Comment by yuhuUSTC** [comment]
/bin/bash train.sh and python train.py (#config .yml file correctly in train.py) are both ok

**Comment by Bruce-WangGF** [comment]
Think you, I have fixed the bugs, Your project has too many document,&nbsp;do those documents apply to earlier comparisons?
I simplify the project based on paper and your project,
Before the image is fed into the network，it will be process with random rotation and flip，and then compute the loss, right?
I don't seem to see a weight of 0.1 of the loss in the project.


If you can, could you please take a look at my repository about the PSDGN reappearance, is there anything missing besides pre-processing of image and gradient descent strategy?
&nbsp;


一梦丶
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年8月1日(星期二) 下午4:28
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [yuhuUSTC/FSDGN] About train (Issue #10)





 
/bin/bash train.sh and python train.py (#config .yml file correctly in train.py) are both ok
